flask_rebar/swagger_generation/marshmallow_to_swagger.py

```diff
@@ -318,14 +328,6 @@ class ValidatorConverter(MarshmallowConverter):
     MARSHMALLOW_TYPE = Validator
 
 
-class DisallowExtraFieldsConverter(SchemaConverter):
-    MARSHMALLOW_TYPE = DisallowExtraFieldsMixin
-
-    @sets_swagger_attr(sw.additional_properties)
-    def get_additional_properties(self, obj, context):
-        return False
```

Comment:
When was this added, and did it ever work?

If it has ever actually done anything, we might need to make it a noop, and just make sure not to break peoples usage of the class and cause them to refactor it on a patch version bump.

Reply:
`DisallowExtraFieldsMixin` has been around for roughly forever, it worked with Rebar 1.x.
It *should* have been removed before we released 2.0.0 with the change from Marshmallow 2 to 3.
It has never actually functioned in "2.x", which was why I went the "sneak this in as 2.0.1 even though it's technically a breaking change" route.  If somebody had a reference to this class and upgraded to 2.0 their code would have already broken like @jab 's 😂

Reply:
ok, sounds good to me. nuke it

Reply:
![](https://c.tenor.com/ZHYmI4BTTWIAAAAC/nuclear-explosion.gif)